src/main/scala/com/microsoft/partnercatalyst/fortis/spark/sources/streamfactories/RSSStreamFactory.scala

```diff
@@ -17,24 +17,17 @@ class RSSStreamFactory extends StreamFactoryBase[RSSEntry] with Loggable {
     import ParameterExtensions._
 
     val params = connectorConfig.parameters
-    connectorConfig.parameters.get("feedUrls") match {
```

Comment:
The `feedUrls` parameter is no longer used, so no sense in having the entire configuration depend on its presence.